TESTING.md

```diff
@@ -28,15 +28,17 @@ Following workflow is intended:
 
 ## Manual Testing
 
-For manual testing with the elastic stack and the agents there is a test environment based on docker. To
-get this running execute the following commands.
-
-* Run `make start-env`
-* Run 3 times around your table, it takes ES quite a bit to startup completely with X-Pack
-* Go to `localhost:5000` or `localhost:5000/error` and press refresh a few times
-* Open `localhost:5601` in your browser and log into Kibana with `elastic` and `changeme`
-* Create the `apm-server-*` index pattern
-* In Kibana go to the Discovery tab and you should see data
-
-For manual testing with specific agents, check instructions at `tests/agent/[LANG]/README.md`
-
+For manual testing with the elastic stack there is a test environment based on docker. 
+To get this running execute the following commands.
+
+* Run `make start-env` to start docker containers for the elastic stack
```

Comment:
This command is outdated, right?

Reply:
ah wait, I see you changed the docker-compose bellow... let me check it out then 👍